bin/oneoff/data_fix/time_spent_milliseconds_to_seconds

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+# The time_spent field in the user_levels table has been
+# recording time in milliseconds. This will cause time_spent
+# to max out at ~25 days. Instead we want to record time in
+# seconds. This will cause time_spent to max out at ~68 years
+#
+# Relevant entries include:
+# SELECT *
+# FROM user_levels
+# WHERE time_spent IS NOT NULL
+#   AND time_spent > 0;
+#
+# time_spent should be transformed as such
+# (time_spent.to_f/1000).ceil.to_i
+
+# upper limit: 2811329000;
+# lower limit:   93221000;
+
+slice = 0
+
+UserLevel.find_in_batches(start: 93_221_000, finish: 2_811_329_000, batch_size: 5000) do |user_level_slice|
+  puts "PROCESSING: slice #{slice} with starting id #{user_level_slice.first.id}."
+  ActiveRecord::Base.transaction do
```

Comment:
I don't think we need to wrap each slice of 5000 rows in a single transaction. Can we log any row that we fail to update (Rescue any error and log the `id` and the reason) and then move on the next row?